### PR TITLE
[desktop] make appstream data pass validation

### PR DESCRIPTION
**Pull request by fgaz**
#535, https://github.com/flathub/flathub/pull/2968#issuecomment-1088370241

### CODE REVIEW

**Comment by fgaz** [comment]
Let's wait for the checks to be successful in the flathub pr before merging

File: qt/res/app.organicmaps.desktop.metainfo.xml [hunk 1]
```diff
@@ -104,4 +105,26 @@
       <image>https://organicmaps.app/images/screenshots/dark.jpg</image>
     </screenshot>
   </screenshots>
+
+  <releases>
+   <release  version="2022.03.23-4-android" date="2022-03-23">
```
Comment (fgaz) [inline qt/res/app.organicmaps.desktop.metainfo.xml:110]: Is this the correct version name? Or should it be just "2022.03.23-4"?

**Comment by fgaz** [comment]
Flathub succeeded (https://github.com/flathub/flathub/pull/2968#issuecomment-1096399602), so this is now ready for review

File: qt/res/app.organicmaps.desktop.metainfo.xml (hunk 1 above)
Comment (biodranik) [reply qt/res/app.organicmaps.desktop.metainfo.xml:110]: Is it an abstract version or a reference to the tag in the repo?

Comment (fgaz) [reply qt/res/app.organicmaps.desktop.metainfo.xml:110]: It's an abstract version

Comment (biodranik) [reply qt/res/app.organicmaps.desktop.metainfo.xml:110]: Then you can remove the -android part.

Comment (fgaz) [reply qt/res/app.organicmaps.desktop.metainfo.xml]: Done

**Review by biodranik — APPROVED** [review]

**Pull request merged**